### PR TITLE
[JBPM-9567] Stabilize Kafka Consumer/Producer tests

### PR DESCRIPTION
**Pull request by gmunozfe**
**JIRA**: [JBPM-9567](https://issues.redhat.com/browse/JBPM-9567)

### CODE REVIEW

File: kie-spring-boot/kie-spring-boot-samples/kie-server-spring-boot-sample/src/test/java/org/kie/server/springboot/samples/KafkaFixture.java [hunk 1]
```diff
@@ -194,13 +208,44 @@ public KModuleDeploymentUnit setup(DeploymentService ds, String artifactId) {
...
+        if (ds!=null && unit!=null) {
             ds.undeploy(unit);
         }
+        if (kieServicesClient != null) {
+            kieServicesClient.disposeContainer(artifactId);
+        }
         System.clearProperty(MESSAGE_MAPPING_PROPERTY);
         System.clearProperty(SIGNAL_MAPPING_PROPERTY);
     }
+
```
Comment (fjtirado) [inline kie-spring-boot/kie-spring-boot-samples/kie-server-spring-boot-sample/src/test/java/org/kie/server/springboot/samples/KafkaFixture.java:222]: I think you can make use of RuntimeDataService Collection<ProcessInstanceDesc> getProcessInstances(List<Integer> states, String initiator, QueryContext queryContext); method to retrieve all active process instance, so there is not need to use rest api

Comment (gmunozfe) [reply kie-spring-boot/kie-spring-boot-samples/kie-server-spring-boot-sample/src/test/java/org/kie/server/springboot/samples/KafkaFixture.java]: Thanks @fjtirado! I haven't seen it, changed

**Review by mareknovotny — APPROVED** [review]
lgtm now, thanks @gmunozfe

**Comment by mareknovotny** [comment]
Could you @fjtirado @afalhambra approve this or comment the latest changes?

**Review by fjtirado — APPROVED** [review]

**Pull request merged**